providers/shared/components/lambda/id.ftl

```diff
@@ -223,6 +223,11 @@
                 "Types" : NUMBER_TYPE,
                 "Default" : -1
             },
+            {
+                "Names" : "ProvisionedExecutions",
```

Comment:
Should have a description to make it clear in the documentation what this does.

Reply:
Added